### ISSUE TITLE
Disk space requirements - better docs

### ISSUE THREAD

**Issue by MarekM25** [NethermindEth/nethermind#4344]
Outdated info: https://docs.nethermind.io/nethermind/ethereum-client/sync-modes

Better info: https://docs.nethermind.io/nethermind/first-steps-with-nethermind/hardware-requirements 

It will be good to add expected chain growth speed.

**Comment by MarekM25** [comment]
Add hardware requirements for full pruning

**Comment by smartprogrammer93** [comment]
> Add hardware requirements for full pruning

@MarekM25  Do we know the requirements?

**Comment by MarekM25** [comment]
> > Add hardware requirements for full pruning
> 
> @MarekM25 Do we know the requirements?

No, the goal of this task is to test and prepare correct values for users
